Gemfile.lock

```diff
@@ -347,7 +347,7 @@ DEPENDENCIES
   webpacker (~> 5.0)
 
 RUBY VERSION
-   ruby 2.7.5p203
+   ruby 2.7.7p221
 
 BUNDLED WITH
-   2.3.12
+   2.4.6
```

Comment:
Have we checked that this version will be compatible with the deploy hosts? I know it was an issue on AWS but I don't know if it matters with Heroku.

Reply:
It was an issue as ansible wasn't updated